hooks/pre-push.d/eof-newline-test

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+top=`git rev-parse --show-toplevel`
+status=0
+for file in $(find . -path ./target -prune -o -type f -name "*.rs" -o -name "*.toml"); do
```

Comment:
```suggestion
for file in $(find . -path ./target -prune -o -type f -name "*.rs" -o -type f -name "*.toml"); do
```

I think we need `-type f` again after the or.

Reply:
Good catch. Fixed in latest push.